src/components/article/index.js

```diff
@@ -42,14 +47,16 @@ export class ViewArticle extends Component {
     loading: false,
   };
 
-  componentDidMount() {
+  async componentDidMount() {
     this.setState({ loading: true });
     const {
       viewArticle: viewSingleArticle,
       getAllArticles: articles,
       match: { params },
+      geter,
+      highlight,
     } = this.props;
-    viewSingleArticle(params.slug);
+    await viewSingleArticle(params.slug, geter, highlight);
```

Comment:
Why are you making this asynchronous? All promised for getting the server data are supposed to be handled in the actions

Reply:
Already using heroku; Let me look into the async issue

Reply:
All sorted